avocado/core/safeloader.py

```diff
@@ -62,18 +62,33 @@ def iter_classes(self):
 
             # Looking for a 'import avocado'
             elif isinstance(statement, ast.Import):
-                for name in statement.names:
-                    if name.name == 'avocado':
-                        if name.asname is not None:
-                            self.mod_imports.add(name.asname)
-                        else:
-                            self.mod_imports.add(name.name)
+                imp_name = statement_import_as(statement).get('avocado', None)
+                if imp_name is not None:
+                    self.mod_imports.add(imp_name)
```

Comment:
The code before add to `mod_imports` only a string (`name.asname` or `name.name`). Now it's adding a `dict` (the result of `statement_import_as`). Isn't that a problem? And if it is, it shouldn't brake any selftest?

Reply:
Good point, it's clearly a mistake... and it looks like the tests covering this should have caught that indeed.  I'll double check them.

Reply:
Actually, no...  `statement_import_as()` returns a dict, but then theres a `.get()` on that dict, returning the name and setting that to `imp_name`.